apps/passport-server/src/services/devconnect/organizerSync.ts

```diff
@@ -409,16 +397,6 @@ export class OrganizerSync {
       const { eventID } = event;
       const approvedLegalTermsEmails: Set<string> = new Set();
 
-      if (this.enableRedaction) {
-        const usersApprovingPII = await fetchUsersByMinimumAgreedTerms(
-          this.db,
-          UNREDACT_TICKETS_TERMS_VERSION
-        );
-        for (const user of usersApprovingPII) {
-          approvedLegalTermsEmails.add(user.email);
-        }
-      }
-
```

Comment:
didn't want to think about how to make it compatible, and it's no longer used. may want to delete all of the old devconnect code, not just this functionality.

Reply:
Deleting everything that is no longer used seems like a good idea to me.

Reply:
I've deleted as much as I think i can get away with.